docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -526,3 +526,13 @@ spec:
   query: SELECT id, name, image, image_id, state, status FROM docker_containers WHERE state = "running";
   purpose: Informational
   contributors: DominusKelvin
+---
+apiVersion: v1
+kind: query
+spec:
+  name: Get Mac and Linux machines with unencrypted primary disks.
```

```suggestion
  name: Get Mac and Linux machines with unencrypted primary disks
```
Can you please remove the period `.` from the end of the query name? This way, the query's name is consistent with the other queries. Thanks :)

---

Done